content/about/grants-faq/contents.lr

```diff
@@ -16,6 +16,7 @@ A: The costs covered by the Grant award may include some or all of the below:
 * Lodging: hotels, motels, hostels, AirBnB, etc. for the days you attend the conference. Reimbursement is limited to the shared cost if the room has more than one occupant.
 * Food (Per Diem): covers food during the days you attend the conference plus a travel day at the beginning and end of your trip at a fixed rate. Please do not upload food receipts. Per-diems are automatically granted to you based on the days that you are attending PyCascades, plus up to one travel day before and after conference attendance.
 * Visa fees: if you are a non US national and must have a visa to attend the conference.
+* Ticket cost: in-person volunteers can request a free ticket or a refund on the ticket cost.
```

Comment:
Maybe this should be anyone can request a free ticket?

Reply:
Yes, we don't limit free tickets based on role